src/rust-crypto/rust-crypto.ts

```diff
@@ -1649,6 +1649,7 @@ function rustEncryptionInfoToJsEncryptionInfo(
     if (shieldState.message === null) {
         shieldReason = null;
     } else if (shieldState.message === "Encrypted by an unverified user.") {
+        // this case isn't actually used with lax shield semantics.
```

Comment:
this is unrelated, but something I came across while I was figuring out what happened and it doesn't feel like worth making a separate PR for